Encryptor/Encryptor.php

```diff
@@ -64,7 +66,12 @@ public function decrypt(string $data): string
         $ivSize     = openssl_cipher_iv_length($this->method);
         $iv         = mb_substr($data, 0, $ivSize, '8bit');
         $cipherText = mb_substr($data, $ivSize, null, '8bit');
+        $decrypt    = openssl_decrypt($cipherText, $this->method, $this->secret, OPENSSL_RAW_DATA, $iv);
```

Comment:
We should also handle encryption

Reply:
I didn't understand, here it's `$decrypt` and already handle, no ?

Reply:
I mean `openssl_encrypt` can also return `false`, so we should fix it as well (so in `encrypt` method)